src/advanced/consensus/specification.md

```diff
@@ -74,8 +74,8 @@ message processing.
 - `propose_timeout`  
   Proposal timeout after a new block is committed to the blockchain locally.
 
-- `round_timeout`  
-  Interval between algorithm rounds.
+- `first_round_timeout`  
+  Initial interval between algorithm rounds at specific height.
```

Comment:
Nit: "at specific height" looks slightly confusing; maybe, it could make sense to use "...at a new blockchain height" or simply omit it?

Reply:
I think it should be mentioned that round timeouts start from initial value after block commit. But I'll try to rephrase it as you suggested.